### PR TITLE
qa/tasks/cephfs/test_scrub: fix self.assertEqual no attribute '_type_equality_funcs'

### PR DESCRIPTION
**Pull request by lxbsz**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by lxbsz** [comment]
The integration test case passed: http://pulpito.ceph.com/xiubli-2020-04-24_06:48:53-fs:basic_functional-wip-fs-misc-2020-04-24-1310-distro-basic-smithi/4979740/

File: qa/tasks/cephfs/test_scrub.py [hunk 1]
```diff
@@ -109,6 +110,9 @@ def validate(self):
 class TestScrub(CephFSTestCase):
     MDSS_REQUIRED = 1
 
+    def setUp(self):
+        super(TestScrub, self).setUp()
```
Comment (tchaikov) [inline qa/tasks/cephfs/test_scrub.py:114]:
```py
super().setUp()
```

Comment (lxbsz) [reply qa/tasks/cephfs/test_scrub.py:114]: Okay, will fix them all.

Comment (lxbsz) [reply qa/tasks/cephfs/test_scrub.py]: Done. Thanks.

**Comment by lxbsz** [comment]
The integration test passed: http://pulpito.ceph.com/xiubli-2020-04-24_08:20:25-fs:basic_functional-wip-lxb-fs-2020-04-24-1447-distro-basic-smithi/

The job id is 4980698.

**Pull request merged**